virtcontainers/mount.go

```diff
@@ -326,3 +326,69 @@ func bindUnmountAllRootfs(ctx context.Context, sharedDir string, sandbox *Sandbo
 		}
 	}
 }
+
+const (
+	dockerVolumePrefix = "/var/lib/docker/volumes"
```

Comment:
It would be great to check where is defined this is docker code just to add as a comment reference in case it changes in the future.

Reply:
`var/lib/docker` is configurable at docker startup I may check only from volumes